src/lib/y2network/connection_config/wireless.rb

```diff
@@ -44,7 +44,7 @@ class Wireless < Base
       # @return [Integer] default WEP key
       attr_accessor :default_key
       # @return [String]
-      attr_accessor :nick
+      attr_accessor :nick # TODO: what it is? identity?
```

Comment:
It is the station node. See "man iwconfig", seems only some diagnostic tools really use it.

Reply:
no, we do not use it. Maybe remove it completely?

Reply:
Yes, please (if possible) I would prefer to drop it.